SQL/New_patches/2020-03-26_add_account_request_date_to_users_table.sql

```diff
@@ -0,0 +1,14 @@
+-- add a new column to the users table to store the date at which
+-- the user requested an account
+ALTER TABLE users
+  ADD COLUMN account_request_date DATE DEFAULT NULL;
+
+-- determine the account_request_date based on what is present in the history
+-- table for when the user was inserted the first time in the history table
+UPDATE users u, history h 
```

Comment:
@ridz1208 is the assumption that all past users requested an account and therefore the date could be derived from the history table correct?

This was the case for Open PREVENT-AD but it might not necessarily be the case for all....